target/linux/mediatek/dts/mt7981b-wavlink-wl-wn586x3.dts

```diff
@@ -0,0 +1,234 @@
+// SPDX-License-Identifier: (GPL-2.0 OR MIT)
+
+/dts-v1/;
+
+#include "mt7981.dtsi"
+
+/ {
+	model = "WAVLINK WL-WN586X3";
+	compatible = "wavlink,wl-wn586x3", "mediatek,mt7981";
+
+	aliases {
+		led-boot = &led_status_blue;
+		led-failsafe = &led_status_blue;
+		led-running = &led_status_blue;
+		led-upgrade = &led_status_blue;
+		serial0 = &uart0;
+	};
+
+	chosen {
+		stdout-path = "serial0:115200n8";
+	};
+
+	gpio-keys {
+		compatible = "gpio-keys";
+
+		button-reset {
+			label = "reset";
+			linux,code = <KEY_RESTART>;
+			gpios = <&pio 11 GPIO_ACTIVE_LOW>;
+		};
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_wan_blue: led@5 {
```

Comment:
I can't find the references to `led_wan_blue:` label in your dts. So it can be dropped. And I agree with @musashino205 (https://github.com/openwrt/openwrt/pull/13783#discussion_r1372512881): `led-0`, `led-1` are more ususal.